components/store/StoreCard.js

```diff
@@ -51,14 +56,41 @@ function StoreCard({ store, callBack, seeProduct }) {
           </Body>
         </InLineContainer>
       )}
-      <InLineContainer style={{ alignItems: 'center' }}>
-        <FontAwesome5
-          name="directions"
-          size={16}
-          color={Colors.secondaryText}
-        />
-        <Body color={Colors.secondaryText}> {address}</Body>
-      </InLineContainer>
+      <TouchableOpacity
+        onPress={() =>
+          Alert.alert(
+            name,
+            address,
+            [
+              {
+                text: 'Get Directions',
+                onPress: () =>
+                  openMap({
```

Comment:
I tested this on my phone (iOS) but because I had my default Maps app deleted, it prompted me to download the default app and is not able to have it open on other map apps (GoogleMaps, Waze etc.). Am not sure if this will be a big problem (I might be the rare few who deletes the default Map app) but just to note the edge case! 

As a reference, the `react-native-open-map` package (https://www.npmjs.com/package/react-native-open-map) might give some additional flexibility if we want to account for this case.

Reply:
WOW thank you `react-native-map-link`(what that's based off of) ended up being exactly what I needed! Wish I had found this earlier.